### PR TITLE
Guess Who Week 5

### PR DESCRIPTION
**Pull request by MariaThomasson**
https://guesswhoavengers.netlify.app/

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -3,62 +3,106 @@
...
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Guess Who Avenger</title>
     <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link rel="shortcut icon" href="favicon.ico" /> <!--https://www.favicon.cc/-->
```
Comment (Rosanna86) [inline code/index.html:12]: Small things that just adds that little extra to the project!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I really got stuck and struggled with the filter functions and arrays. First I used true and false and the game worked fine but after wednesday class I wanted to change it into []. Then I only got errors and undifined messeges.. But after talking to one of my fellow Zebras we solved my problems using a code-solution she came up with that also worked for me. 
 
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://guesswhoavengers.netlify.app/
+
+Guess Who bubble was made by me in Canva. https://www.canva.com
```
Comment (Rosanna86) [inline README.md:20]: Really nice!

Comment (Rosanna86) [inline README.md:20]: And wow what a great thing!

File: code/index.html [hunk 3]
```diff
@@ -3,62 +3,106 @@
...
-        <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
-        </optgroup>
-        <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+
+      <!---- Drop down menu default start ---->
```
Comment (Rosanna86) [inline code/index.html:37]: Good with all the comments! Makes it easier for me as a code reviewer to read the code.

File: code/index.html [hunk 4]
```diff
@@ -3,62 +3,106 @@
...
+
+      <option selected disabled default>Select</option>
+   
+      <!---- Drop down menu content ---->
+
+                <!---- Hair ---->
+        <optgroup label="hair color">
```
Comment (Rosanna86) [inline code/index.html:44]: Small thing - but maybe write the headline for each attribute with capital first letter? Hair, What, Accessories and Abilites?

File: code/style.css [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
+  --primary: #152238;
+  --secondary: #1c2e4a;
 }
 
 body {
   display: flex;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Bangers', cursive;
```
Comment (Rosanna86) [inline code/style.css:11]: Love the font! Excellent for your chosen theme

File: code/style.css [hunk 6]
```diff
@@ -137,6 +147,7 @@ button {
 .outlined-button:hover {
   background-color: var(--primary);
   color: white;
+  text-shadow: 0px 0px 6px rgba(255,255,255,0.7);
```
Comment (Rosanna86) [inline code/style.css:150]: Nice touch on the Find Out button with the glowing effect

File: code/style.css [hunk 7]
```diff
@@ -17,38 +18,48 @@ h1 {
...
   min-height: 100vh;
-  background-color: var(--secondary);
+  background-color: #cccc;
   display: flex;
   flex-direction: column;
-  padding: 5vw;
+  padding: 3vw;
   align-items: flex-start;
   box-sizing: border-box;
+  background-image: url("images/game.jpg");
```
Comment (Rosanna86) [inline code/style.css:33]: Love this image!

File: code/script.js [hunk 8]
```diff
@@ -229,97 +231,191 @@ const setSecret = () => {
...
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+// Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  setSecret()
+  generateBoard()
+  selectQuestion()
+
+  console.log(secret)
```
Comment (Rosanna86) [inline code/script.js:240]: console.log can be removed when project is finished :)

File: code/script.js [hunk 9]
```diff
@@ -229,97 +231,191 @@ const setSecret = () => {
...
+    stringPropertyName = 'whatSpecies'
     if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all people with yellow hair"
+      alert(
+        `Yes, the Avenger is a ${value}! Keep all Avengers that is a ${value}`
+      )
     } else {
-      // alert popup that says something like: "No, the person doesnt have yellow hair! Remove all people with yellow hair"
+      alert(
+        `No, the Avenger isn´t a ${value}! Remove all Avenger that isn´t a ${value}`
```
Comment (Rosanna86) [inline code/script.js:333]: Small typo: Avengers

File: code/script.js [hunk 10]
```diff
@@ -229,97 +231,191 @@ const setSecret = () => {
...
-      charactersInPlay = charactersInPlay.filter((person) => person[attribute] === value)
-      or
-      charactersInPlay = charactersInPlay.filter((person) => person[attribute] !== value)
-
-    for accessories and other
-      charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value))
-      or
-      charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
-  */
+  else if (category === 'abilities') {
```
Comment (Rosanna86) [inline code/script.js:349]: So happy that my simplifying of the code helped you. I can say that with out writing out everything in this way I hadn't understand one bit of this part of the code.

I think in the beginning of our journey to become a front end developer this is the way to do it. Just to understand what each bit of code are doing.

**Review by Rosanna86 — COMMENTED** [review]
You did a great job Maria! Starting the week great by creating a new game board, then you came across some issues but managed to pull everything together for Fridays presentation. 

Easy code to read with all the comments. Great design and you managed to get a grip of this weeks javascript. 

Blue Level (Minimum Requirements) [YES]
Red Level (Intermediary Goals) [YES, some of the steps]

File: code/index.html (hunk 1 above)
Comment (MariaThomasson) [reply code/index.html:12]: Thank you!

File: code/index.html (hunk 3 above)
Comment (MariaThomasson) [reply code/index.html:37]: Thanks :)

File: code/index.html (hunk 4 above)
Comment (MariaThomasson) [reply code/index.html:44]: Good point!

File: code/script.js (hunk 8 above)
Comment (MariaThomasson) [reply code/script.js:240]: That right, I will remove it :)

File: code/script.js (hunk 9 above)
Comment (MariaThomasson) [reply code/script.js:333]: Thank you for noticing! Will change it.

File: code/script.js (hunk 10 above)
Comment (MariaThomasson) [reply code/script.js:349]: So thankful for this, made my week!! And I totally agree.

**Pull request closed without merging**